src/main/resources/application.properties

```diff
@@ -2,4 +2,5 @@ springdoc.swagger-ui.path=/swagger-ui
 cors-allowed-origin = http://localhost:5173/
 spring.jpa.hibernate.ddl-auto=update
 spring.datasource.driver-class-name=org.h2.Driver
+app.isProd=false

```

Comment:
application.properties is for all profiles, so you need to override it in application-prod.properties. You can test the prod environment by adding the "prod" profile in the run/debug configurations.

Reply:
done